Tensile/SolutionLibrary.py

```diff
@@ -31,6 +31,12 @@
 from .SolutionStructs import Solution as OriginalSolution
 from .Utils import state
 
+architectureList = ['fallback', 'gfx803', 'gfx900', 'gfx906', 'gfx908', 'gfx90a',
+                    'gfx940', 'gfx941', 'gfx942', 'gfx1010', 'gfx1011', 'gfx1012',
+                    'gfx1030', 'gfx1031', 'gfx1032', 'gfx1034', 'gfx1035', 'gfx1100', 
+                    'gfx1101', 'gfx1102']
+solutionIndexMap = {architectureName:int(offset*pow(2,16)) 
+                    for architectureName,offset in zip(architectureList,range(len(architectureList)))}
```

Comment:
While you debug you might just want to use the gfx number as is hex for the high 16bits.   Solution indices aren't going to be preserved across release I thought so if there is ever trouble you can drop down to sequence int.

Reply:
Do you mean that you prefer the upper 16 bits to be, for example, int(0x90a, 16) right shifted 16 bits for gfx90a?

Reply:
The idea was just while analyzing and debugging you don't have to look at that order list to figure out the sequence table to gfx number from high 4 bytes, you can just look at it in hex.